### PR TITLE
2 get topics

### PR DESCRIPTION
**Pull request by zajk9999**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+			.get("/api/topics")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.topics.length).toBe(3);
+				body.topics.forEach((topic) => {
+					expect(typeof topic.slug).toBe("string");
+					expect(typeof topic.description).toBe("string");
+				});
+			});
+	});
```
Comment (nc-jo-watson) [inline __tests__/app.test.js:27]: This is great! ⭐

File: app.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const express = require("express")
+const { getTopics } = require("./controllers/nc-news-controllers")
+const app = express()
+
+app.get("/api/topics", getTopics)
+
+
+// error handling
```
Comment (nc-jo-watson) [inline app.js:8]: Comments shouldn't be in the code that lives on the `main` branch, so remember to remove them before doing a PR 😄

File: models/nc-news-models.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection")
+
+exports.fetchTopics = () => {
+    return db
+    .query("SELECT * FROM topics;")
+    .then(({rows}) => {
+        if(!rows.length) {
+            return Promise.reject({status: 404, msg: "Not Found"})
+        }
```
Comment (nc-jo-watson) [inline models/nc-news-models.js:9]: Two questions for you....
1. Do you need this error? Generally a 404 is returned when we've requested a specific record or a subset of records in the database (E.g. I want all the coffee related topics, but there are no coffee topics! ☕ ). But here we just want all the topics from the database.
2. Where does this error go 👀  You have not caught it anywhere, and you don't have any tests for it either...

**Review by nc-jo-watson — COMMENTED** [review]
Really nice work Łukasz! Remember to remove your comments before a PR/merging to main, and have a little think about your error handling here.

File: __tests__/app.test.js (hunk 1 above)
Comment (zajk9999) [reply __tests__/app.test.js:27]: Thank you Jo!

File: models/nc-news-models.js (hunk 3 above)
Comment (zajk9999) [reply models/nc-news-models.js:9]: Thank's for spotting this. I actually had it caught, passed and handled. But then I realised, that with this endpoint it makes no sense and I ended up deleting it and obviously I forgot to remove it from the model. Will do of course :) and I will do my best to remember to get rid of comments ;)

**Pull request merged**